### PR TITLE
Adds capability to edit user's lat lng  data

### PR DESCRIPTION
**Pull request by mango-habanero**

### CODE REVIEW

File: app/server/utils/user.py [hunk 1]
```diff
@@ -444,16 +444,18 @@ def proccess_create_or_modify_user_request(
                             or attribute_dict.get('payment_card_barcode'))
 
     location = attribute_dict.get('location')  # address location
-    geo_location = attribute_dict.get('geo_location')  # geo location as str of lat, lng
+    lat = attribute_dict.get('latitude')
+    lng = attribute_dict.get('longitude')
 
+    """geo_location = attribute_dict.get('geo_location')  # geo location as str of lat, lng
```
Comment (nolash) [inline app/server/utils/user.py]: please remove commented code

File: app/client/components/user/EditUserForm.tsx [hunk 2]
```diff
@@ -33,6 +33,8 @@ export interface IEditUser {
   bio?: string;
   referredBy?: string;
   location?: string;
+  latitude?: number;
```
Comment (nolash) [inline app/client/components/user/EditUserForm.tsx:36]: forgive my ignorance; does `number` cover `float` too?

**Review by nolash — COMMENTED** [review]
There is no test code with this PR. Please verify that unit test for `app/server/model/user` actually sets all the respective fields correctly in db, or expand test to cover that, optionally also functional test for the function in `app/server/utils`

File: app/client/components/user/EditUserForm.tsx (hunk 2 above)
Comment (mango-habanero) [reply app/client/components/user/EditUserForm.tsx:36]: Yes it does.
[Number in typescript](https://www.typescriptlang.org/docs/handbook/basic-types.html#number)

**Comment by mango-habanero** [comment]
> There is no test code with this PR. Please verify that unit test for `app/server/model/user` actually sets all the respective fields correctly in db, or expand test to cover that, optionally also functional test for the function in `app/server/utils`

A test to this effect already exists:

```
def test_create_transfer_account_user(
        test_client, init_database, create_master_organisation,
        last_name, location, lat, lng, initial_disbursement):
    from flask import g
    g.active_organisation = create_master_organisation

    assert proccess_create_or_modify_user_request(
        attribute_dict=dict(
        first_name='Lilly', last_name=last_name,
        phone=fake.msisdn(),
        location=location, lat=lat, lng=lng,
        initial_disbursement=initial_disbursement))
```

**Review by nolash — DISMISSED** [review]

**Comment by nolash** [comment]
blocked by https://github.com/GrassrootsEconomics/SempoBlockchain/pull/32

**Comment by nolash** [comment]
Work will be re-submitted in gitlab.

**Pull request closed without merging**